### PR TITLE
Build assets with rollup for prod with umd output

### PR DESCRIPTION
**Pull request by bashmach**
## Changes

- use rollup for dev and prod
- add `umd` output option

### CODE REVIEW

**Comment by pipedrive-bot-eventsink** [comment]
### Main branch dependencies
[snyk vulnerabilities](https://app.snyk.io/org/gravity-byw/project/5013564b-fbd2-498f-ad71-ca01748463df)

File: rollup.config.js [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
-};
+export default [
+	{
+		input: 'src/index.ts',
+		output: [
+			{
+				file: 'dist/index.js',
+				format: 'cjs',
+				sourcemap: true,
+				exports: 'named',
```
Comment (AndreasSaltsberg) [inline rollup.config.js:12]: Thoughts on adding terser plugin here as well?

**Review by AndreasSaltsberg — APPROVED** [review]
LGTM 🏆

**Comment by pipedrive-bot-dora** [comment]
## Neodora Analysis ⚠


<details><summary><strong> ⚠ Missing cache in Github Workflow</strong></summary>
<br/>

**Message:** Github workflow should be using caching when step uses `actions/setup-node`

<table>
<thead>
	<tr>
			<th>Workflow file</th>
			<th>Job name</th>
			<th>Step index</th>
			<th>Actual cache value</th>
			<th>Expected cache value</th>
	</tr>
</thead>
<tbody>
		<tr>
				<td><a href="https://github.com/pipedrive/app-extensions-sdk/tree/75797589a152b09e9be6e74245590dd3c278ed33/.github/workflows/on-commit.yml">on-commit.yml</a></td>
				<td><code>lint</code></td>
				<td>2</td>
				<td><code>Not found</code></td>
				<td><code>npm</code></td>
		</tr>
</tbody>
</table>


🔍 [How to fix it](https://neodora.pipedrive.tools/rules/workflowCache)
<br/>
</details>

<details><summary><strong> ⚠ Unknown tribe name</strong></summary>
<br/>

**Message:** The property `owner` in `repository.json` must be a valid tribe name present in **mission-tracking**
**Repository file:** [repository.json](https://github.com/pipedrive/app-extensions-sdk/tree/75797589a152b09e9be6e74245590dd3c278ed33/repository.json)

<table>
<thead>
	<tr>
			<th>Owner status</th>
	</tr>
</thead>
<tbody>
		<tr>
				<td>'GRAVITY' is an unknown tribe name</td>
		</tr>
</tbody>
</table>


🔍 [How to fix it](https://neodora.pipedrive.tools/rules/tribeName)
<br/>
</details>


Commit [`7579758`](https://github.com/pipedrive/app-extensions-sdk/commit/75797589a152b09e9be6e74245590dd3c278ed33/)  Job [`49398`](https://jenkins.pipedrive.tools/job/neodora/49398/)

**Pull request merged**